integration_tests/e2e/appointments/createVideoLinkCourtHearing.cy.ts

```diff
@@ -0,0 +1,186 @@
+import getCategories from '../../fixtures/activitiesApi/getAppointmentCategories.json'
+import getMdiAppointmentLocations from '../../fixtures/prisonApi/getMdiAppointmentLocations.json'
+import getScheduledEvents from '../../fixtures/activitiesApi/getScheduleEvents-MDI-A1350DZ-A8644DY.json'
+import getScheduledEventLocations from '../../fixtures/activitiesApi/getScheduledEventLocations.json'
+import HowToAddPrisonersPage from '../../pages/appointments/create-and-edit/howToAddPrisonersPage'
+import ReviewPrisonersPage from '../../pages/appointments/create-and-edit/reviewPrisonersPage'
+import { formatDate } from '../../../server/utils/utils'
+import ExtraInformationPage from '../../pages/appointments/create-and-edit/extraInformationPage'
+import getPrisonPrisoners from '../../fixtures/prisonerSearchApi/getPrisonPrisoners-MDI-A8644DY.json'
+import getPrisonerA8644DY from '../../fixtures/prisonerSearchApi/getPrisoner-MDI-A8644DY.json'
+import getAppointmentDetails from '../../fixtures/activitiesApi/getAppointmentDetails.json'
+import ReviewPrisonerAlertsPage from '../../pages/appointments/create-and-edit/reviewPrisonerAlertsPage'
+import getCourtList from '../../fixtures/bookAVideoLinkApi/getCourtList.json'
+import getCourtHearingTypes from '../../fixtures/bookAVideoLinkApi/getCourtHearingTypes.json'
+import getBvlsLocations from '../../fixtures/bookAVideoLinkApi/getBvlsLocations.json'
+import getBvlsVccRoom1 from '../../fixtures/bookAVideoLinkApi/getBvlsLocation-VCC_ROOM_1.json'
+import getCompletedCourtBooking from '../../fixtures/bookAVideoLinkApi/getCompletedCourtBooking.json'
+import getPrisonerAlerts from '../../fixtures/alertsApi/getPrisonerAlertsA8644DY.json'
+import PrisonLocationsPage from '../../pages/appointments/create-and-edit/video-link-booking/prisonLocationsPage'
+import HearingDetailsPage from '../../pages/appointments/create-and-edit/video-link-booking/hearingDetailsPage'
+import VideoLinkDateAndTimePage from '../../pages/appointments/create-and-edit/video-link-booking/videoLinkDateAndTimePage'
+import CourtHearingLinkPage from '../../pages/appointments/create-and-edit/video-link-booking/courtHearingLinkPage'
+import VideoLinkSchedulePage from '../../pages/appointments/create-and-edit/video-link-booking/videoLinkSchedulePage'
+import VideoLinkCourtCheckAnswersPage from '../../pages/appointments/create-and-edit/video-link-booking/videoLinkCourtCheckAnswersPage'
+import VideoLinkConfirmationPage from '../../pages/appointments/create-and-edit/video-link-booking/videoLinkConfirmationPage'
+import { YesNo } from '../../../server/@types/activities'
+
+context('Create video link court appointment', () => {
+  const tomorrow = addDays(new Date(), 1)
+  const tomorrowFormatted = formatDate(tomorrow, 'yyyy-MM-dd')
+
+  // To pass validation we must ensure the appointment start date is set to tomorrow
+  getScheduledEvents.activities
+    .filter(e => e.prisonerNumber === 'A7789DY')
+    .forEach(e => {
+      e.prisonerNumber = 'A8644DY'
```

Comment:
I don't understand what this is doing with prisoner number(s).  They are different to each other (same for the probation journey).

Reply:
Good question. I've tidied this and added a comment - its not strictly needed, and does not impact the test, but it does add other events to the prisoners schedule so that the user can see other things happening on the day of the booking.